ui/main/src/app/services/card.service.ts

```diff
@@ -58,16 +58,16 @@ export class CardService {
 
     private selectedCardId: string = null;
 
-    private receivedAcksSubject = new Subject<{cardUid: string, entitiesAcks: string[]}>();
-    private receivedDisconnectedSubject = new Subject<boolean>()
+    private receivedAcksSubject = new Subject<{cardUid: string, cardId: string, entitiesAcks: string[]}>();
```

Comment:
you do not need the cardId , i do not see any use in the code

Reply:
Done.